app/scripts/modules/core/help/helpContents.js

```diff
@@ -527,4 +527,14 @@ module.exports = angular.module('spinnaker.core.help.contents', [])
     'markdown.examples': 'Some examples of markdown syntax: <br/> *<em>emphasis</em>* <br/> **<b>strong</b>** <br/> [link text](http://url-goes-here)',
     'application.enableRestartRunningExecutions': 'When this option is enabled, users will be able to restart pipeline stages while a pipeline is still running. ' +
     'This behavior can have varying unexpected results and is <b>not recommended</b> to enable.',
+    'pipeline.config.webhook.payload': 'Json payload to be added to the webhook call.',
+    'pipeline.config.webhook.waitForCompletion': 'If not checked, we consider the stage succeeded if the webhook return an HTTP status code 2xx, otherwise it will be failed. If checked, it will poll a status url (defined below) to grasp the progress of the stage.',
```

Comment:
Please s/return/returns

Also, 'grasp the progress of the stage' sounds off to me - how about 'determine the progress of the stage'?